.github/workflows/pr-review-notify.yml

```
@@ -0,0 +1,122 @@
+name: PR Review Comment Notifier
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)